services/web/client/source/class/osparc/utils/LibVersions.js

```diff
@@ -119,6 +119,19 @@ qx.Class.define("osparc.utils.LibVersions", {
       }, this);
 
       return libs;
+    },
+
+    getPlatformName: function() {
+      const stackName = qx.core.Environment.get("osparc.stackName");
+      let platformName = "dev";
+      if (stackName.includes("master")) {
+        platformName = "master";
+      } else if (stackName.includes("staging")) {
+        platformName = "staging";
+      } else if (stackName.includes("production")) {
+        platformName = "";
```

Comment:
then why not use "production" since you add it anyway in services/web/client/source/class/osparc/auth/LoginPage.js ?

Reply:
(explained above)
because I am directly using the ouptut of that for the little label we are now printing by the osparc logo